gneiss/util.py

```diff
@@ -184,3 +188,81 @@ def rename_internal_nodes(tree, names=None, inplace=False):
             n.name = label
             i += 1
     return _tree
+
+
+def _intersect_of_table_metadata_tree(table, metadata, tree):
+    """ Matches tips, features and samples between the table, metadata
```

Comment:
I don't think I can get this to fit within 1 line.  Do you think it is really necessary for a helper method?